sdk/containerregistry/azure-containers-containerregistry/CHANGELOG.md

```diff
@@ -4,10 +4,21 @@
 
 ### Features Added
 
-### Breaking Changes
-
-- Replaced `uploadManifest(OciImageManifest)` method on `ContainerRegistryBlobClient` and `ContainerRegistryBlobAsyncClient` classes
-  with `uploadManifest(OciImageManifest manifest, String tag)` method.
+### Breaking Changes from version 1.1.0-beta.3
+
+- `ContainerRegistryContentClientBuilder`, `ContainerRegistryBlobClient` and `ContainerRegistryBlobAsyncClient` were renamed to `ContainerRegistryContentClientBuilder`,
+  `ContainerRegistryContentClient`, and `ContainerRegistryContentAsyncClient` and moved to `com.azure.containers.containerregistry` package. 
+- `UploadBlobResult` was renamed to `UploadRegistryBlobResult`
```

Comment:
In .NET I changed DownloadBlobResult to DownloadRegistryBlobResult as well to match.  The full diff is here: [APIView](https://apiview.dev/Assemblies/Review/7ebfcb36ff6149ba96170bd059094513/4a92edb1e3cb4247bba9c61e345b9ff6?diffRevisionId=f989159861d846b3abb4384d01a27da9&doc=False&diffOnly=True)

Reply:
I don't have DownloadBlobResult - I'm following storage here and BinaryData or write to stream directly. I think we had a conversation about it at some point and it's a specific pattern that you have in .NET where you don't return BinaryData directly?